### ISSUE TITLE
Distance: Toggle quantitatve/qualitative distance description infront of helptext with toggle of Distance checkbox (title changed))

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#101]
In addition to #100, please update the first sentence of the Sphere Position help text paragraph. The first sentence is a dynamic distance description of distance and I'd like it to include the word "centers" toggle with the Distance check box. 

This first sentence needs to toggle between quantitative (distance checked) and qualitative (distance not checked) descriptions for distance. This is working in the descriptions of the Mass Spheres, and it also needs to happen here.

**When Distance is checked use the quantitative distance in kilometers:**
-  Centers of spheres, {{{{4}} kilometers apart}}}}. Move spheres closer or farther from each other.

**When Distance is NOT checked use the qualitative described distance (same as in state-of-the-sim list):**
-  Centers of spheres, {{{{close to}} each other}}}}. Move spheres closer or farther from each other.

**Comment by terracoda** [comment]
@zepumph, I updating the previous comment and the issue title to be match the screenshots shown in issue #100.

**Comment by terracoda** [comment]
Screen shot showing this change is in #100.

**Comment by zepumph** [comment]
Implemented above. I updated the code so the REGULAR and BASICS were using the same PDOM Node for the `ul` that help the sphere position sliders in the PDOM. 

I then updated the BASICS help text to toggle qualitative/quantitative descriptions depending on if distance values are showing.

There was also a bit more refactoring to reuse a string piece that looks like `{{close to}} each other` within the `PositionDescriber` class. Over to you @terracoda to review.
